multisrc/overrides/madara/cerisescans/src/CeriseScans.kt

```diff
@@ -16,7 +16,7 @@ import java.util.concurrent.TimeUnit
 
 class CeriseScans : Madara(
     "Cerise Scans",
-    "https://cerisescan.com",
+    "https://cerisescans.com/",
```

Comment:
Remove the trailing slash.

```suggestion
    "https://cerisescans.com",
```